va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
long format: application parse slice header and send slice parameter to driver, the num_ref_idx_active_override_flag is true by default.
short format: application will not parse slice header, it send slice header+slice data to driver directly. slice header will be parsed by driver